### PR TITLE
chore: fixed some of the terms language

### LINKED ISSUE #13333: [CAL-2977] show privacy and terms
null

<sub>[CAL-2977](https://linear.app/calcom/issue/CAL-2977/show-privacy-and-terms)</sub>

### PR DESCRIPTION
**Pull request by PeerRich**

### CODE REVIEW

File: apps/web/pages/settings/organizations/[id]/onboard-admins.tsx [hunk 1]
```diff
@@ -20,6 +20,8 @@ const OnboardTeamMembersPage = () => {
         title={t("invite_organization_admins")}
         description={t("invite_organization_admins_description")}
       />
+      <div className="mt-8 h-[414px]" />
+      <iframe className="mx-auto h-full w-[276px]" src="https://cal.games/" />
```
Comment (PeerRich) [inline apps/web/pages/settings/organizations/[id]/onboard-admins.tsx:24]:
```suggestion
```

**Review by Udit-takkar — APPROVED** [review]
LGTM. Fixed the Trans component bug

Screenshot 2024-01-19 at 10 09 13 PM

Screenshot 2024-01-19 at 10 09 26 PM

**Comment by ciaranha** [comment]
Were going to put it on the left 

image

**Comment by PeerRich** [comment]
> Were going to put it on the left 
> 
> image

will make this change

also in weekly and column, we'll just put it above the bottom left mini-calendar

**Pull request merged**